specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json

```diff
@@ -347,22 +347,36 @@
       }
     },
     "Appearance": {
-      "description": "An object representing the style and styleConfidence.",
+      "description": "An object representing the appearance of the text line.",
       "type": "object",
       "required": [
-        "style",
-        "styleConfidence"
+        "style"
       ],
       "properties": {
         "style": {
-          "description": "The text line style.",
+          "description": "An object representing the Style of the text line.",
+          "type": "object",
+          "$ref": "#/definitions/Style"
+        }
+      }
+    },
+    "Style": {
+      "description": "An object representing the style of the text line.",
+      "type": "object",
+      "required": [
+        "name",
+        "confidence"
+      ],
+      "properties": {
+        "name": {
+          "description": "The text line style name.",
           "type": "string",
           "enum": [
```

Comment:
Need to add x-ms-enum.modelAsString=true.  See https://github.com/Azure/azure-rest-api-specs/blob/aa4bbdafcabcf4726a8ccbe2ab529cb4fe48ad6f/specification/cognitiveservices/data-plane/ComputerVision/preview/v3.1-preview.2/Ocr.json#L441 for an example.

http://azure.github.io/autorest/extensions/#x-ms-enum

Reply:
Thanks!